src/test/java/edu/hawaii/its/api/service/TestGroupPathService.java

```diff
@@ -74,16 +74,6 @@ public void isValidPath() {
         assertFalse(groupPathService.isValidPath("bad-path"));
     }
 
-    @Test
-    public void isGroupingPath() {
-        assertTrue(groupPathService.isGroupingPath(GROUPING));
-        assertFalse(groupPathService.isGroupingPath(GROUPING_BASIS));
-        assertFalse(groupPathService.isGroupingPath(GROUPING_OWNERS));
-        assertFalse(groupPathService.isGroupingPath(GROUPING_INCLUDE));
-        assertFalse(groupPathService.isGroupingPath(GROUPING_EXCLUDE));
-        assertFalse(groupPathService.isGroupingPath(INVALID_GROUPING_PATH));
-    }
```

Comment:
Make sure to keep this test in, thanks!

Reply:
Should we actually keep this test in, since there is no longer isGroupoingPath(String) function in the grouping path? There is isGroupingPath(Group) function though.

Reply:
Yes, and I just noticed the original isGroupingPath(String) is removed too. I would just keep that for consistency with having a isGroupPath(String). The old isGroupingPath function is still actually accurate because it makes a call to check if the grouping exists in Grouper.

Reply:
Someone just told me that the implementation of the old function was not correct, and it shouldn't be used.